packages/server/lib/controllers/auth/postApiKey.ts

```diff
@@ -90,7 +90,7 @@ export const postPublicApiKeyAuthorization = asyncWrapper<PostPublicApiKeyAuthor
                 : await logContextGetter.create(
                       {
                           operation: { type: 'auth', action: 'create_connection' },
-                          meta: { authType: 'apikey', connectSession: endUserToMeta(res.locals.endUser) },
+                          meta: { authType: 'apikey', connectSession: res.locals.endUser ? endUserToMeta(res.locals.endUser) : undefined },
```

Comment:
should the condition be in `endUserToMeta`?

Reply:
yeah right I can change that too 👌🏻